app.yaml

```diff
@@ -11,10 +11,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-runtime: custom
+runtime: ruby
 vm: true
 api_version: 1
 module: default
+# [START entrypoint]
+entrypoint: bundle exec foreman start --formation "$FORMATION"
+# [END entrypoint]
 
 resources:
```

Comment:
I see.  For hello world - I'd like to keep app.yaml as minimal as possible.  In the nodejs getting started we really just use the two fields (runtime and vm) and kick you out to the docs page:

https://cloud.google.com/nodejs/getting-started/hello-world